src/System.Data.SqlClient/tests/ManualTests/SQL/TcpDefaultForAzureTest/TcpDefaultForAzureTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Diagnostics;
+using Xunit;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    public static class TcpDefaultForAzureTest
+    {
+        private const string NP = "Named Pipes Provider";
+        private const string TCP = "TCP Provider";
+        private static string[] azureExtensions =
+                {
+                    ".database.windows.net",
+                    ".database.cloudapi.de",
+                    ".database.usgovcloudapi.net",
+                    ".database.chinacloudapi.cn"
+                };
+
+        [Fact]
+        public static void ConnectByTcpIfAzureTest()
```

Comment:
This test should be refactored into two tests.
One which checks with Azure Endpoints and one without Azure.